pkg/kubelet/cm/deviceplugin/manager.go

```diff
@@ -106,10 +165,20 @@ func (m *ManagerImpl) CheckpointFile() string {
 	return filepath.Join(m.socketdir, kubeletDevicePluginCheckpoint)
```

Comment:
Maybe rename this function to `checkpointFile` ?

Reply:
Good idea - getting that in.